app/state_machines/flood_risk_engine/work_flow.rb

```diff
@@ -6,7 +6,7 @@ module FloodRiskEngine
   class WorkFlow
     # Define each work flow within Definitions
     module Definitions
-      extend self
+      module_function
```

Comment:
New way of doing this for me. `module_function` looks neater than `extend self`, and the fact it's in Ruby Core suggests this is the expected technique to use. 

However, worth looking at the [docs for `module_function`](http://ruby-doc.org/core-2.3.0/Module.html#method-i-module_function) as I don't think the overriding of the method behaves as I would expect.